lib/internal/Magento/Framework/File/Uploader.php

```diff
@@ -132,16 +132,6 @@ class Uploader
      */
     const TMP_NAME_EMPTY = 666;
 
-    /**
-     * Max Image Width resolution in pixels. For image resizing on client side
-     */
-    const MAX_IMAGE_WIDTH = 1920;
-
-    /**
-     * Max Image Height resolution in pixels. For image resizing on client side
-     */
-    const MAX_IMAGE_HEIGHT = 1200;
```

Comment:
Constants cannot be removed, see http://devdocs.magento.com/guides/v2.0/contributor-guide/backward-compatible-development/. Just deprecate them adding `@see` section.